opencti-platform/opencti-front/src/utils/filters/useSearchEntities.js

```diff
@@ -372,6 +373,7 @@ const useSearchEntities = ({
                 label: defaultValue(n.node),
                 value: n.node.id,
                 type: n.node.entity_type,
+                parentTypes: n.node.parent_types,
```

Comment:
Why is this different from the Contains filter ?

Reply:
I don't understand the question

Reply:
Parent_types is the proper way of doing this, we need to align the other filters to this way